src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/JSImportExportTest.cs

```diff
@@ -36,6 +36,7 @@ public async Task MultipleImportAsync()
             instance1.Dispose();
         }
 
+#if !FEATURE_WASM_THREADS // because in MT JSHost.ImportAsync is really async, it will finish because the caller could cancel
```

Comment:
I think this is meant to be something like `the operation will finish before before the caller can cancel it`

it's a shame i can't think of a good way to test this with threads active